python/setup.cfg

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[metadata]
+name = py-iceberg
+version = 0.1.0
+author = Apache Software Foundation
+author_email = dev@iceberg.apache.org
+description = Apache Iceberg is an open table format for huge analytic datasets.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://iceberg.apache.org/
+project_urls =
+    Source Code = https://github.com/apache/iceberg/
+    Issue Tracker = https://github.com/apache/iceberg/issues
+license="Apache License 2.0",
+license_files =
+    LICENSE
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+package_dir =
+    = src
```

Comment:
These two lines don't look correct to me.

Reply:
Based on https://packaging.python.org/tutorials/packaging-projects/#configuring-metadata, 
```
package_dir is a mapping of package names and directories. An empty package name represents the “root package” — the directory in the project that contains all Python source files for the package — so in this case the src directory is designated the root package.
```
This is equivalent to `package_dir={"": "src"},` if written in `setup.py`.